### PR TITLE
Feature/using computers heathily refresh

### PR DESCRIPTION
**Pull request by Blizzardsev**
Introduces the following:

- Overhaul for the `talk_using_computers_healthily` topic, including support for revisiting the topic, activity system and revised dialogue style.

### CODE REVIEW

File: game/script-topics.rpy [hunk 1]
```diff
@@ -1044,61 +1044,238 @@ init 5 python:
...
+    extend 4tsgbg " just make sure you're sitting about an arm's length away from the screen.{w=0.75}{nw}"
+    extend 4ccsbg " That's all I'm saying!"
+    n 3unmpu "Don't forget to keep all your stuff in easy reach though -{w=0.5}{nw}"
+    extend 3ccsbg " unless you're a psychic,{w=0.5}{nw}"
+    extend 6csqsm " you aren't getting far if you can barely hit the keyboard."
+    n 7ccsbg "Now,{w=0.2} number three:{w=0.75}{nw}"
+    extend 3fchbg " breaks!"
+    n 3tllfl "I don't know about you [player],{w=0.5}{nw}"
+    extend 7tnmbo " but personally?{w=0.75}{nw}"
+    extend 7csrem " I can't stand being stuck in one place for hours at a time."
```
Comment (RaionArt) [inline game/script-topics.rpy:1197]: ngl that's a bit ironic given her situation. I could imagine that would encourage players even more to demand new places to visit.

File: game/script-topics.rpy [hunk 2]
```diff
@@ -1044,61 +1044,238 @@ init 5 python:
...
+    extend 3ullaj " then you should have your arms and thighs parallel to the ground,{w=0.5}{nw}"
+    extend 3fcssm " with your eyes roughly at the top of your screen.{w=0.75}{nw}"
+    extend 3fcsbg " Easy peasy!"
+    n 4unmfl "Oh,{w=0.2} right -{w=0.5}{nw}"
+    extend 2clrsssbl " make sure your feet can still touch the floor,{w=0.2} though.\n{w=0.75}{nw}"
+    extend 2fcscasbr "Even {i}I{/i} can do that,{w=0.2} [player]."
+    n 1fnmaj "Number two:{w=0.75}{nw}"
+    extend 2fcsbg " distance!"
+    n 2clrbg "It's pretty easy to forget,{w=0.5}{nw}"
+    extend 4tnmbo " but if you wanna avoid sore eyes then you've gotta make sure you're actually a decent distance from the screen too."
```
Comment (RaionArt) [inline game/script-topics.rpy:1170]: Could maybe add to look out the window to let the eyes focus on larger distance than just the closed up desktop.

File: game/script-topics.rpy [hunk 3]
```diff
@@ -1044,61 +1044,238 @@ init 5 python:
...
+        n 2fcstr "A-{w=0.2}and besides,{w=0.2} [player]."
+        n 2clrajsbl "It's not like I'm gonna get mad at you or anything like that either if you gotta duck out.{w=0.75}{nw}"
+        extend 2clrsll " Even I'm not that much of a jerk."
+        n 1clrbol "..."
+
+    n 4ccsflsbr "A-{w=0.2}anyway."
+    n 4ullaj "I've gone on way long enough already,{w=0.5}{nw}"
+    extend 2cnmca " so I'm just gonna say this,{w=0.2} [player]."
+    n 2fcsaj "You might end up with a crappy back or nasty eyes if you aren't careful..."
+    n 3fcspo "...But that's gonna be nothing compared to the grade-A earache you're getting if you come crying to me later!"
```
Comment (RaionArt) [inline game/script-topics.rpy:1268]: With the different `arms_crossed` poses it looks like she is clipping through the desk here.

**Review by RaionArt — CHANGES_REQUESTED** [review]
A nice rework to fit more in the modern writing of JN! I actually enjoy that NatNat is making a difference of the activity of the player now. Overall the conversation has a better flow to it and more unique ways of how Natsuki talks. Only added some comments for tweaks!

File: game/script-topics.rpy (hunk 1 above)
Comment (Blizzardsev) [reply game/script-topics.rpy:1197]: True; I can reword it to emphasise her stance on sitting specifically (as she at least gets up from her desk when the player isn't around, or goes to the closet to fetch things)

File: game/script-topics.rpy (hunk 3 above)
Comment (Blizzardsev) [reply game/script-topics.rpy:1268]: Good catch, I'll adjust

File: game/script-topics.rpy (hunk 2 above)
Comment (Blizzardsev) [reply game/script-topics.rpy:1170]: Sure, I can add a segment for this

**Comment by Blizzardsev** [comment]
Adjustments made

**Review by RaionArt — APPROVED** [review]
Thank you for your time and adding these adjustments!
